src/scss/_brand.scss

```diff
@@ -69,6 +80,10 @@
 			shaded-close-text: oColorsGetPaletteColor('white'),
 			shaded-close-border: oColorsGetPaletteColor('white')
 		),
-		'supports-variants': ()
+		'supports-variants': (
+			'compact',
```

Comment:
As all brands support all variants, is this and `_oOverlaySupports('...')` needed?